website/events/static/events/css/admin.scss

```diff
@@ -48,7 +48,7 @@
 
         select[name=payment] {
             background: var(--message-error-bg);
-            color: white;
+            color: var(--text-color);
```

Comment:
This technically does not work due to the variable not existing. However, it appears because the default colour for form element is `var(--body-fg)`. So we can just remove this property.

```suggestion
```

Reply:
Wot hahah, my IDE said it did exist. But sure